static/app/views/performance/traceDetails/index.tsx

```diff
@@ -23,6 +23,7 @@ import withApi from 'sentry/utils/withApi';
 import withOrganization from 'sentry/utils/withOrganization';
 
 import TraceDetailsContent from './content';
+import TraceDetailsContentV0 from './contentV0';
```

Comment:
I don't think it's clear to see `V0` in component names, nor does this tell me which component is the first or the new one. Is there a way we can differentiate these better?

In the worst case we could fall back to something like `NewTraceDetailsContent` but if that's our only option since this is temporary

Reply:
Renamed them to be more detailed. It's a bit chonky but it's temporary so we shudn't hurt much